integration-test/test.js

```diff
@@ -16,6 +16,12 @@ describe('in afterEach mode', function () {
     expect.addAssertion('<function> when run through (mocha|jest) <assertion>', (expect, subject) => {
         expect.errorMode = 'nested';
         const isMocha = expect.alternations[0] === 'mocha';
+
+        if (process.env.JEST === 'false' && expect.alternations[0] === 'jest') {
+            // Allow to disable jest assertions when running integration tests for coverage.
+            return expect(true, 'to be ok');
+        }
+
```

Comment:
I don't understand this part?

Reply:
nyc is unable to collect coverage from the tests running in jest, so I just made the jest variations a noop when running the coverage script.

Our travis script will run the following tests:

- all tests in the `test`-folder in both mocha and jest
- all tests in the `integration-test`-folder in both mocha and jest
- coverage for tests in both folders in just mocha

Currently we don't have any difference in implementation between the two test runners. So I didn't want to go the extra mile to gather coverage from both runners and merge them together before submitting and reporting. That is an option though as the coverage formats are the same/similar enough IIRC.

Reply:
Ah, so it could just be `return;` as well? The assertion was the thing that confused me the most :)

Reply:
I assumed I had to return an assertion for it not to trip up :-) I admittedly did not test it with a plain return statement.